web/src/app/alerts/CreateAlertDialog/StepContent/step1.js

```diff
@@ -0,0 +1,196 @@
+import {
+  Grid,
+  TextField,
+  InputAdornment,
+  List,
+  ListItem,
+  ListItemText,
+  ListItemIcon,
+  Paper,
+  Typography,
+  InputLabel,
+  Chip,
+} from '@material-ui/core'
+import { makeStyles, emphasize } from '@material-ui/core/styles'
+import { FormField } from '../../../forms'
+import ServiceLabelFilterContainer from '../../../services/ServiceLabelFilterContainer'
+import { Search as SearchIcon } from '@material-ui/icons'
+import FavoriteIcon from '@material-ui/icons/Star'
+import { ServiceChip } from '../../../util/Chips'
+import AddIcon from '@material-ui/icons/Add'
+
+const useStyles = makeStyles(theme => ({
+  addAll: {
+    backgroundColor: theme.palette.grey[100],
+    height: theme.spacing(3),
+    color: theme.palette.grey[800],
+    fontWeight: theme.typography.fontWeightRegular,
+    '&:hover, &:focus': {
+      backgroundColor: theme.palette.grey[300],
+      textDecoration: 'none',
+    },
+    '&:active': {
+      boxShadow: theme.shadows[1],
+      backgroundColor: emphasize(theme.palette.grey[300], 0.12),
+      textDecoration: 'none',
+    },
+  },
+  chipContainer: {
+    padding: theme.spacing(0.5),
+    marginBottom: theme.spacing(2),
+    height: '9em',
+    overflow: 'auto',
+    border: '1px solid #bdbdbd',
+  },
+  endAdornment: {
+    display: 'flex',
+    alignItems: 'center',
+  },
+  noticeText: {
+    width: '100%',
+    textAlign: 'center',
+    alignSelf: 'center',
+    lineHeight: '9em',
+  },
+}))
+
+const CREATE_ALERT_LIMIT = 35
+
+export default props => {
+  const { formFields, queriedServices } = props
+
+  const fieldRef = useRef()
+  const classes = useStyles()
+
+  const labelKey = formFields.searchQuery.split(/(!=|=)/)[0]
+  const labelValue = formFields.searchQuery
+    .split(/(!=|=)/)
+    .slice(2)
+    .join('')
+
+  const AddAll = () => (
+    <Chip
+      component='button'
+      label='Add All'
+      disabled={formFields.selectedServices.length >= CREATE_ALERT_LIMIT}
+      icon={<AddIcon fontSize='small' />}
+      onClick={() => {
+        const toAdd = queriedServices.map(s => s.id)
+
+        // build newState
+        let newState = formFields.selectedServices
+        toAdd.forEach(s => {
+          if (newState.length < CREATE_ALERT_LIMIT) {
+            newState = newState.concat(s)
+          }
+        })
+
+        props.onChange({ selectedServices: newState })
+      }}
+      className={classes.addAll}
+    />
+  )
+
+  const serviceChips = formFields.selectedServices.map((id, key) => {
+    return (
+      <ServiceChip
+        key={key}
+        clickable={false}
+        id={id}
+        style={{ margin: 3 }}
+        onClick={e => e.preventDefault()}
+        onDelete={() =>
+          props.onChange({
+            selectedServices: formFields.selectedServices.filter(
+              sid => sid !== id,
+            ),
+          })
+        }
+      />
+    )
+  })
+
+  const notice = (
+    <Typography variant='body1' component='p' className={classes.noticeText}>
```

Comment:
This should probably be a caption in the top left for consistency, with a text color of `textSecondary`, which is what we use for info text.

e.g.
```
<Typography color='textSecondary' variant='caption'>
  Visit this escalation policy for more information.
</Typography>
```